galaxy/normalization/NmrNormalization_wrapper.R

```diff
@@ -126,7 +131,7 @@ if (graphique != "None")
 ##-------
   # Data
 data_normalized <- cbind(rownames(data_normalized),data_normalized)
-colnames(data_normalized) <- c("Bucket",colnames(data_normalized)[-1])
+colnames(data_normalized) <- c("Variable",colnames(data_normalized)[-1])
```

Comment:
You have to update your test-data because you change your output header
See [here](https://travis-ci.org/workflow4metabolomics/normalization/builds/335073388#L2194)